pkg/event/ptp/event.go

```diff
@@ -12,25 +12,11 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package hwevent
+package ptp
 
 const (
-	// TextPlain ...
-	TextPlain = "text/plain"
-	// TextJSON ...
-	TextJSON = "text/json"
-	// ApplicationJSON ...
-	ApplicationJSON = "application/json"
+	// EVENT is the event type for ptp event
+	EVENT = "com.redhat.cne.ptp_event"
```

Comment:
replace redhat  and type is from api spect , known type

Reply:
can you add another constant called source
and add this to it
```
//State of equipment synchronization is notified at state change
/sync/sync-status/sync-state

//ptp-state-change notification is signalled from equipment at state change
/sync/ptp-status/lock-state

//Notification used to inform about synce synchronization state change
/sync/synce-status/lock-state

//gnss-state-change notification is signalled from equipment at state change
/sync/gnss-status/gnss-sync-status

// State of node OS clock synchronization is notified at state change
/sync/sync-status/os-clock-sync-state

//synce-state change notification is signalled from equipment at state change, enhanced information

/sync/synce-status/lock-state-extended

//ptp-clock-class-change notification is generated when the clock-class changes.  
event.sync.ptp-status.ptp-clock-class-change 

//synce-clock-quality-change notification is generated when the clock-quality changes.  
/sync/synce-status/clock-quality

```

Reply:
done